src/AWS.Deploy.Orchestration/Utilities/DeployedApplicationQueryer.cs

```diff
@@ -246,6 +269,20 @@ private async Task<List<CloudApplication>> GetExistingBeanstalkEnvironments()
                 optionSettings[tuple.OptionSettingId] = configurationSetting.Value;
             }
 
+            if (recipeId.Equals(Constants.RecipeIdentifier.EXISTING_BEANSTALK_WINDOWS_ENVIRONMENT_RECIPE_ID))
+            {
+                var manifestPath = Path.Combine(Path.GetDirectoryName(projectPath) ?? string.Empty, Constants.ElasticBeanstalk.WindowsManifestName);
+                if (_fileManager.Exists(manifestPath))
+                {
+                    var manifest = JsonSerializer.Deserialize<ElasticBeanstalkWindowsManifest>(await _fileManager.ReadAllTextAsync(manifestPath));
+                    if (manifest.Deployments.AspNetCoreWeb.Count != 0)
```

Comment:
Curious - why if this is a list is it okay to just take the first? Could you add a comment here?

Reply:
This was based on the implementation the toolkit was using and also the windows beanstalk recipe we already have out. That and all the examples of the manifest file online only had 1 entry. I checked the documentation and there wasn't a lot of info on what other entries could be added.